### PR TITLE
Updated Creation time due to incorrect casting. 

### PR DESCRIPTION
**Pull request by NotIntense**
Caused the messages in the screenshots shown below. Adjusted from a cast to correctly subtracting total time from current time.

image
image

### CODE REVIEW

File: Exiled.API/Features/Ragdoll.cs [hunk 1]
```diff
@@ -141,8 +141,12 @@ public bool CanBeCleanedUp
         /// <summary>
         /// Gets the time that the ragdoll was spawned.
         /// </summary>
-        public DateTime CreationTime => new((long)NetworkInfo.CreationTime);
-
+        public DateTime CreationTime
+        {
+            get => DateTime.Now - TimeSpan.FromSeconds(NetworkInfo.CreationTime);
+            set => NetworkInfo.CreationTime = value;
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Ragdoll.cs:147]:
```suggestion
            set
            {
                float creationTime = (float)(NetworkTime.time - (DateTime.Now - value).TotalSeconds);
                NetworkInfo = new RagdollData(NetworkInfo.OwnerHub, NetworkInfo.Handler, NetworkInfo.RoleType, NetworkInfo.StartPosition, NetworkInfo.StartRotation, NetworkInfo.Nickname, creationTime);
            }
```

File: Exiled.API/Features/Ragdoll.cs [hunk 2]
```diff
@@ -141,8 +141,12 @@ public bool CanBeCleanedUp
         /// <summary>
         /// Gets the time that the ragdoll was spawned.
         /// </summary>
-        public DateTime CreationTime => new((long)NetworkInfo.CreationTime);
-
+        public DateTime CreationTime
+        {
+            get => DateTime.Now - TimeSpan.FromSeconds(NetworkInfo.CreationTime);
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Ragdoll.cs:146]:
```suggestion
            get => DateTime.Now - TimeSpan.FromSeconds(NetworkInfo.ExistenceTime);
```

File: Exiled.API/Features/Ragdoll.cs [hunk 3]
```diff
@@ -141,8 +141,12 @@ public bool CanBeCleanedUp
         /// <summary>
         /// Gets the time that the ragdoll was spawned.
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Ragdoll.cs:142]:
```suggestion
        /// Gets or sets the time that the ragdoll was spawned.
```

**Comment by joker-119** [comment]
Fix build errors.

File: Exiled.API/Features/Ragdoll.cs [hunk 4]
```diff
@@ -139,10 +139,18 @@ public bool CanBeCleanedUp
...
+        public DateTime CreationTime
+        {
+            get => DateTime.Now - TimeSpan.FromSeconds(NetworkInfo.ExistenceTime);
+            set
+            {
+                float creationTime = (float)(NetworkTime.time - (DateTime.Now - value).TotalSeconds);
+                NetworkInfo = new RagdollData(NetworkInfo.OwnerHub, NetworkInfo.Handler, NetworkInfo.RoleType, NetworkInfo.StartPosition, NetworkInfo.StartRotation, NetworkInfo.Nickname, creationTime);
+            }
+        }
+
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Ragdoll.cs:153]:
```suggestion

```

File: Exiled.API/Features/Ragdoll.cs [hunk 5]
```diff
@@ -139,10 +139,17 @@ public bool CanBeCleanedUp
...
-
+        public DateTime CreationTime
+        {
+            get => DateTime.Now - TimeSpan.FromSeconds(NetworkInfo.ExistenceTime);
+            set
+            {
+                float creationTime = (float)(NetworkTime.time - (DateTime.Now - value).TotalSeconds);
+                NetworkInfo = new RagdollData(NetworkInfo.OwnerHub, NetworkInfo.Handler, NetworkInfo.RoleType, NetworkInfo.StartPosition, NetworkInfo.StartRotation, NetworkInfo.Nickname, creationTime);
+            }
+        }
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Ragdoll.cs:152]:
```suggestion
        }

```

Comment (Misaka-ZeroTwo) [reply Exiled.API/Features/Ragdoll.cs:152]: github web kinda cursed

**Pull request merged**